srcs/runners_manager/vm_creation/gcloud/GcloudManager.py

```diff
@@ -73,6 +105,13 @@ def configure_instance(
         )
         disk_size_gb = runner.vm_type.config["disk_size_gb"]
         disk_type = f"projects/{self.project_id}/zones/{self.zone}/diskTypes/pd-ssd"
+        labels = {}
+        for tag in runner.vm_type.tags:
+            if "small" in tag or "medium" in tag or "large" in tag:
+                labels["flavor"] = tag
+                break
```

Comment:
I understand why you did this, but there's too much risk of it not working or needing to be adapted as our flavor names evolve.

let's make it easier for now and remove this need from the requirement, you can:
- Remove the `flavor` label
- Or/and set the label `machine_type` instead, which should provide the same information

Reply:
Done